tests/rustdoc-gui/tooltips.goml

```diff
@@ -0,0 +1,15 @@
+// This test checks that the right font is applied to the `i` tooltip element.
```

Comment:
I think this should probably mention this is technically a regression test for https://github.com/rust-lang/rust/issues/144778 (and also the PR description should mention this fixes that issue)

Reply:
It's actually not. Likely the same bug but I can't reproduce #144778 so I'll wait for @fmease to confirm that the fix I provide them is working as expected.

Reply:
Resolving conversation then. Thanks for confirming it @fmease!